openshift/template.yaml

```diff
@@ -286,6 +287,58 @@ objects:
                     key: ${JOBS_AUTO_REPORT_SLACK_CHANNEL_ID_KEY}
                     name: ${JOBS_AUTO_REPORT_SLACK_CHANNEL_ID_NAME}
 
+- apiVersion: batch/v1
+  kind: CronJob
+  metadata:
+    name: elasticsearch-cleanup
+  spec:
+    concurrencyPolicy: ${ELASTICSEARCH_CLEANUP_CONCURRENCY_POLICY}
+    schedule: ${ELASTICSEARCH_CLEANUP_SCHEDULE}
+    suspend: ${{ELASTICSEARCH_CLEANUP_SUSPEND}}
+    jobTemplate:
+      spec:
+        ttlSecondsAfterFinished: 10800 # keep last 3 runs (3 * 1h)
```

Comment:
I think this can be parametrized

Reply:
Thank you, I will adjust both comments in a new PR as it should be in all the cronjobs